pyazhpc/arm.py

```diff
@@ -651,7 +654,10 @@ def _add_vm(self, cfg, r, vnet_in_deployment):
                 vmres["properties"]["proximityPlacementGroup"] = {
                     "id": f"[resourceId('Microsoft.Compute/proximityPlacementGroups','{rppgname}')]"
                 }
-            
+
+            if rstoragesku == "UltraSSD_LRS" :
+                vmres["properties"]["additionalCapabilities"] = { "ultraSSDEnabled": "true" }
```

Comment:
is this supposed to be a string or bool value?

Reply:
@hmeiland 
fyi here is the rest api - https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/createorupdate#additionalcapabilities - which looks to be of type bool

Reply:
yep: agree